programs/ssl/ssl_client2.c

```diff
@@ -253,10 +256,26 @@ int main( void )
     "                             This cannot be used with eap_tls=1\n"
 #define USAGE_NSS_KEYLOG_FILE                               \
     "    nss_keylog_file=%%s\n"
-#else
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+#define USAGE_SRTP \
+    "    use_srtp=%%d         default: 0 (disabled)\n" \
+    "                          This cannot be used with eap_tls=1 or "\
+    "                          nss_keylog=1\n"             \
+    "    srtp_force_profile=%%d  default: all enabled\n"   \
```

Comment:
Which numeric value selects this default behaviour - `0`? If so, it might be worth adding that: `default: all enabled (0)`.

Reply:
It should be `default: 0 (all enabled)` for consistency. That's not very important though, the documentation of many options is incomplete anyway.

Reply:
fixed